cvat-core/src/api.ts

```diff
@@ -295,13 +295,30 @@ function build() {
                     return result;
                 },
                 settings: {
-                    async get(taskID: number) {
-                        const result = await PluginRegistry.apiWrapper(cvat.analytics.quality.settings.get, taskID);
+                    async get(filter: any) {
+                        const result = await PluginRegistry.apiWrapper(cvat.analytics.quality.settings.get, filter);
+                        return result;
+                    },
+                    async update(values: any) {
+                        const result = await PluginRegistry.apiWrapper(cvat.analytics.quality.settings.update, values);
+                        return result;
+                    },
+                    async create(values: any) {
+                        const result = await PluginRegistry.apiWrapper(cvat.analytics.quality.settings.create, values);
+                        return result;
+                    },
+                    async defaults() {
+                        const result = await PluginRegistry.apiWrapper(cvat.analytics.quality.settings.defaults);
                         return result;
                     },
                 },
             },
         },
+        scheme: {
+            async get() {
+                return PluginRegistry.apiWrapper(cvat.scheme.get);
+            },
+        },
```

Comment:
It should be in server namespace.

```
cvat.server.scheme.get()
```

Or just ``cvat.server.scheme()`. I  do not really expect that client can do something else with the server  scheme

Reply:
Updated